react/features/display-name/components/DisplayName.web.js

```diff
@@ -144,15 +146,12 @@ class DisplayName extends Component {
             );
         }
 
-        const suffix
-            = displayName && displayNameSuffix ? ` (${displayNameSuffix})` : '';
-
```

Comment:
By the way, this logic was duplicated in ContactListPanel. If you want to fix it now go ahead otherwise I'll do it some time this week.